public_html/wp-content/plugins/camptix-attendance/addons/assets/attendance-ui.scss

```diff
@@ -221,6 +221,20 @@ li.item {
 	display: block;
 	margin-right: 50px;
 	margin-left: 50px;
+
+	.previews {
+		position: fixed;
+		bottom: 1em;
+		left: 1em;
+		right: 1em;
+
+		video {
+			max-height: 8em;
+			max-width: 8em;
+			width: 8em;
+			float: right;
+		}
+	}
```

Comment:
While there were changes in the `.css` file only before, these changes are only in the `.scss` file. It's not clear which library was used to compile the `.css` file, but I'd assume some older Sass version with the default settings.

Reply:
not sure where this is comming from, I'll keep it for now